cmd/api/src/api/v2/saved_queries_test.go

```diff
@@ -47,13 +47,29 @@ func TestResources_ListSavedQueries_SortingError(t *testing.T) {
 	defer mockCtrl.Finish()
 
 	endpoint := "/api/v2/saved-queries"
+	userId, err := uuid2.NewV4()
+	require.Nil(t, err)
+
+	bhCtx := ctx.Context{
+		RequestID: "",
+		AuthCtx: auth.Context{
+			Owner: model.User{
+				Unique: model.Unique{
+					ID: userId,
+				},
+			},
+		},
+		Host: nil,
+	}
+	goContext := bhCtx.ConstructGoContext()
 
 	if req, err := http.NewRequest("GET", endpoint, nil); err != nil {
 		t.Fatal(err)
 	} else {
 		q := url.Values{}
 		q.Add("sort_by", "invalidColumn")
 
+		req = req.WithContext(goContext)
```

Comment:
This makes sense; I set it up this way because that is the pattern that was in place for the tests that were already setting up an Auth context (starting around line 275). Would you like me to update those as well?